### DISCUSSION TITLE
关于播放问题

### DISCUSSION THREAD

**Discussion by 563088655** [Richasy/Bili.Uwp#1678] [问答]
我在播放的时候，经常会点击一个视频之后就直接闪退，请问是运存不足吗？可是我机器16g啊

**Comment by intresed** [comment]
我也是遇到这个问题，我电脑是surface laptop studio 16g,设置为核显运行，有的视频点进去就卡住，然后就闪退了。

**Reply by 563088655** [reply]
我的机器是幻x2022，3050版本

**Reply by intresed** [reply]
那我们显卡配置差不多，我是是xe核显加3050ti 版本，我已经设置成使用集显，但还是闪退，而且同样的视频第二次打开就可能不闪退了，看来是随机的，不知道是什么原因。
